Chaincase.UI/wwwroot/style/global.css

```diff
@@ -156,3 +156,56 @@ body.dark {
 
     --ion-card-background: #1e1e1e;
 }
+
+.ios .transition {
+    margin-top: 44px;
+}
+
+.md .transition {
+
+    margin-top: 60px;
+}
```

Comment:
use padding-top. Margin top pushes the whole content down and clips the bottom. we just want padding inside the element